src/assets/styles/_functions.scss

```diff
@@ -20,3 +20,21 @@
 
   @return $result;
 }
+
+//
+// Replace `$search` with `$replace` in `$string`
+// @author Hugo Giraudel
+// @param {String} $string - Initial string
+// @param {String} $search - Substring to replace
+// @param {String} $replace ('') - New value
+// @return {String} - Updated string
+//
+@function str-replace($string, $search, $replace: '') {
+  $index: str-index($string, $search);
+
+  @if $index {
+    @return str-slice($string, 1, $index - 1) + $replace + str-replace(str-slice($string, $index + str-length($search)), $search, $replace);
```

Comment:
Not sure where you're using this but this seems like a good case for just using a JavaScript library like Emotion to take care of the evaluation of this style.

Reply:
I think this was more over-complication than needed; your comment actually spurred an idea that is much simpler and won't require another function. I'll amend this PR with that.